src/widgets/textlabel.h

```diff
@@ -2,7 +2,6 @@
 #define TEXTLABEL_H
 
 #include <QLabel>
-#include <QMouseEvent>
```

Comment:
which toolchains?  That sounds really weird, since `qwidget.h` gives a forward declaration of `class QMouseEvent`.  Which toolchain can't deal with a pointer to a forward declared class?